src/utils/getPercentageOfDaysLeft.ts

```diff
@@ -1,19 +1,20 @@
 import moment from 'moment';
 
-function inputParser(input: string) {
-    const parsedDate = moment(new Date(parseInt(input, 10) * 1000));
-    return parsedDate;
-}
-
 const getPercentageOfDaysLeft = (startedOn: string, endsOn: string): number => {
-    const startDate = inputParser(startedOn);
-    const endDate = inputParser(endsOn);
+    const startDate = moment(new Date(parseInt(startedOn, 10) * 1000));
+    const endDate = moment(new Date(parseInt(endsOn, 10) * 1000));
 
-    // It provides us with total days that are there for the the project and number of days left
+    /**
+     * It provides us with total days that are there for the the project and
+     * number of days left
+     */
     const totalDays = endDate.diff(startDate, 'days');
     const daysLeft = endDate.diff(new Date(), 'days');
 
-    // It provides the percentage of days left
+    /**
+     * It provides the percentage of days left
+     */
+
```

Comment:
I think you should have defined a documentation comment at the top of the file for one thing.